### ISSUE TITLE
[DB] Northrend herbs

### ISSUE THREAD

**Issue by Knindzagxg** [azerothcore/azerothcore-wotlk#1857]
More and more players (Over 20 now) reporting there is very very small number of herbs spawns in Northrend at the moment.

Made a character and checked my self -> There is so small number of them.

Check it yourself:

Create character
Teach yourself herbalism
Tele anywhere to Icecrown
Set some speed and track flowers
Roam for a while and see how much of them is spawned

Commit: up to date

**Comment by poszer** [comment]
Checking ac database, and comparing them with some wotlk databases it looks fine..

- [ ] Goldclever 207

- [ ] Firethorn 21

- [ ] Tiger Lily 134

- [ ] Talandra's Rose 77

- [ ] Frozen Herb 135

- [ ] Adder's Tongue 84

- [ ] Lichbloom 180

- [ ] Icethorn 142

- [ ] Frost Lotus 4

- [ ] Mana Thistle 35

- [ ] Nightmare Vine 52

- [ ] Netherbloom 68

- [ ] Netherdust Bush 43

- [ ] Ragveil 51

- [ ] Terocone 108

- [ ] Felweed 415

- [ ] Frozen Herb 2

- [ ] Icecap 182

**Comment by poszer** [comment]
Maybe best test for this will be to download that addon for client which show the correct locations of Herbs, Nodes etc , i don't remember name, but i think it's Gatherer than compare with our

**Comment by BarbzYHOOL** [comment]
it's not critical priority but high :)

**Comment by sluggor** [comment]
I looked into this a little, and the issue may just be that the pool size is too small.

All Storm Peaks herbs are going into pool 981, which is set to a max size of 35. If I'm reading the code correctly, that means that of the 129 instances of Lichbloom (190171) and Icethorn (190172), only 35 will be instantiated at a time.

That may be too low for that zone.

By comparison, there are two pools of ore in Storm Peaks totaling 70 instances allowed at once.

@Knindzagxg, can you try a test for me? If you have a database you can test on, run this to increase the pool size for herbs in Storm Peaks and then fly around again and see if it seems better.

```SQL
USE `acore_world`;

UPDATE `pool_template` set `max_limit` = 70 WHERE `entry` = 981;
```

Thanks.

**Comment by Knindzagxg** [comment]
Seems a bit better now sluggor cheers.

I'm not sure how much is supposed to be spawned at once tho, need to dig that info somewhere.

**Comment by BarbzYHOOL** [comment]
ty for testing knindza

**Comment by spinzaku** [comment]
i used @sluggor 's suggestion and increased the pool_template 'max_limit'
fixed my issues on low northrend herb spawn

**Comment by felwyn** [comment]
Fix approved! More herbs spawning now. More than enough for this zone. 👍

**Comment by Si1ker** [comment]
### [  ISSUE RESOLVED ON #3338 ]

* cannot link the PR directly into this issue as the PR is already merged.
